### ISSUE TITLE
[BUG] Players not working properly after exit Fullscreen

### ISSUE THREAD

**Issue by wookje** [sarbagyastha/youtube_player_flutter#158]
**Describe the bug**
I'm trying to make a GirdView with couple of videos. It works perfect(even when I switch videos randomly) on original screen. But once get into a full screen, and get back to the original screen, then the bug occurs.  

Following unexpected situation occurs randomly (occurs together sometimes).
    (1) Controller(play, pause, bottom bar, etc.) disappears but video still goes playing.
    (2) Controller alives but plays audio only, with screen paused.
    (3) Only play, pause buttons alive. Bottom bar disappears. Then, (2) happens to other videos.

**To Reproduce**
Below is my part of code. I'm using `youtube_player_flutter: ^6.0.3+2`

```
_controller = new List<YoutubePlayerController>();
_url = [
  "https://youtu.be/GAJgbyzSsxg",
  "https://youtu.be/93QS6CReUbQ",
  "https://youtu.be/qch5tdUQvYk",
  "https://youtu.be/-7XFvZmIcTE",
  "https://youtu.be/9w7e02i39lM",
  "https://youtu.be/HtNL9-35xaI",
  "https://youtu.be/baHYYZScKaw"
];
for (int i = 0; i < _url.length; i++) {
  YoutubePlayerController tmp = YoutubePlayerController(
    initialVideoId: YoutubePlayer.convertUrlToId(_url[i]),
    flags: YoutubePlayerFlags(autoPlay: false),
  );
  _controller.add(tmp);
}

// ...

child: GridView.count(
  crossAxisCount: 2,
  shrinkWrap: true,
  physics: NeverScrollableScrollPhysics(),
  crossAxisSpacing: 5,
  primary: true,
  children: List.generate(_url.length, (index) {
    return Center(
      child: YoutubePlayer(
        controller: _controller[index],
        onReady: (){},
      ),
    );
  }),
),
```

**Technical Details:**
 - Device: Samsung Galaxy S9+

Thank you for making good plugin :)

**Comment by joelnv** [comment]
@sarbagyastha  i am also facing the same issue, video player after exit of fullscreen i am not able to make the video player fullscreen again.

**Comment by sarbagyastha** [comment]
Should fix with v7.0.0. Feel free to reopen if the issue persists.
